src/__tests__/FormContainer.test.tsx

```diff
@@ -82,6 +86,10 @@ describe('Form container', () => {
       it('should have formMethods setFieldToTouched prop', () => {
         expect(wrappedComponent.props().formMethods).toHaveProperty('setFieldToTouched');
       });
+
+      it('should get run provided middleware', () => {
+        expect(wrappedComponent.props()).toHaveProperty('bar');
+      });
```

Comment:
What about...?

```
expect(wrappedComponent.state('model')).toEqual(initialModel)
```

Reply:
that's middleware, adds bar attribute to the rest of the props.

there's a test for initialModel, too

Reply:
Sorry, I wasn't referencing the middleware test... but I don't see the test for initialModel.

Reply:
this is a good assertion though
expect(wrappedComponent.props().form.model).toEqual(initialModel)